requirements.txt

```diff
@@ -26,6 +26,7 @@ pyyaml~=3.12
 vulture~=0.10.0
 bandit~=1.2
 nbformat>=4.1
+pyflakes~=1.2.0
```

Comment:
OK. Now lets investigate options on how to fix that. 1.2 is *not* an acceptable option.